src/service/viewport/viewport-binding-ios-embed-wrapper.js

```diff
@@ -237,6 +241,22 @@ export class ViewportBindingIosEmbedWrapper_ {
     return rect.height + rect.top + this.getScrollTop();
   }
 
+  /** @override */
+  contentHeightChanged() {
+    if (isExperimentOn(this.win, 'scroll-height-bounce')) {
+      // Refresh the overscroll (`-webkit-overflow-scrolling: touch`) to avoid
+      // iOS rendering bugs. See #8798 for details.
+      const doc = this.win.document;
+      const {documentElement} = doc;
+      this.vsync_.mutate(() => {
+        documentElement.classList.remove('i-amphtml-ios-overscroll');
```

Comment:
per our chat, this won't be effecting if user has 
```
    html {
      -webkit-overflow-scrolling: touch 
    }
```
already set.

should should force it with a:
```
    i-amphtml-ios-overscroll-disable {
      -webkit-overflow-scrolling: auto !important 
    }
```

Reply:
The outer `html` doesn't allow changes to this property. This is why we have `i-amphtml-ios-overscroll` class.